### PR TITLE
various-fixes

### PR DESCRIPTION
**Pull request by JayOrten**
This PR adds various fixes:
- remove quotation marks from config
- add parameters for gradient clipping, betas, and weight decay
- remove accidental slurm script in user_slurms folder
- Saves logging to a csv file, as well as TB, and fixes some of the logging stuff so it's logging correctly. I tested this.
- Fixes the checkpointing naming mechanism. For the normal `.ckpt` files, it will save with the number of steps. The HF checkpoints are still saved based on `num_checkpoints`. This is a bit of a tricky problem because of the way torch takes a format string in order to come up with a unique checkpoint name.  I don't know how important the HF checkpoints are at this point, so I think this fix is okay for now.

### CODE REVIEW

**Review by DrewGalbraith — APPROVED** [review]
I like what you're thinking in here! I'm pickin up what you're puttin down.
